lib/mutant/mutator/node/arguments.rb

```diff
@@ -59,7 +55,7 @@ def emit_mlhs_expansion
           mlhs_childs_with_index.each do |child, index|
             dup_children = children.dup
             dup_children.delete_at(index)
-            dup_children.insert(index, *child.children)
+            dup_children.insert(index, *child)
```

Comment:
Why the splat? Does the `child` respond to `#to_a` or `#to_ary` (can't remember which expands in the presence of the splat arg)? Or can child be `nil`? (doubtful, but figured I should ask)